re_data_ui/src/partials/Graph.tsx

```diff
@@ -30,12 +30,15 @@ type GenerateGraphResponseProps = {
 }
 
 const getAlertData = (modelId: string, aggregatedModels: Map<string, ReDataModelDetails>) => {
-  const {
-    anomalies,
-    schemaChanges,
-  } = aggregatedModels.get(modelId) as ReDataModelDetails;
+  if (aggregatedModels.has(modelId)) {
```

Comment:
@akinolu52 one question regarding that, I believe this fix is no longer needed after changed to mapping?
Maybe we should just revert to the previous version here / which just fails?

Reply:
or maybe we should leave it as it's an extra check & it does no harm @mateuszklimek

Reply:
ok, let's do that ;)